src/page/admin/auth/AutoCreate/AutoCreateForm.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { FormControl, Grid, MenuItem, Switch, TextField, Select, InputLabel } from '@material-ui/core';
+
+function AutoCreateForm({ data = {}, setValue }) {
```

Comment:
Consider keeping the same naming convention on containing folder.

Reply:
would it then make sense to call the component "index.tsx" or would it be better with "AutoCreateForm/AutoCreateForm.tsx"?

Reply:
AutoCreateForm.tsx would be preferred because it is much easier for the VS code search to locate that component when doing text search.

Reply:
fixed